components/slider/src/use-range-slider.ts

```diff
@@ -351,6 +393,7 @@ export const useRangeSlider_unstable = (
     };
   }, [controlRef, handleTouchStart, removeListeners]);
 
+  // Range slider values, *from* and *to* values?
```

Comment:
?

Reply:
I can remove the comment. It just helps me remembering that `minValue` contains from my way of seeing it the start if the range and not the minimum possible or allowed value.

Reply:
Here `minValue` is `selectedMinValue` but could also be falsely read as `allowedMinValue`. Therefore my commend. But maybe we could improve the variable names instead someday.